ts/features/bonus/cgn/screens/CgnDetailScreen.tsx

```diff
@@ -103,100 +178,111 @@ const CgnDetailScreen = (props: Props): React.ReactElement => {
         isLoading(props.unsubscriptionStatus)
       }
     >
-      <BaseScreenComponent
-        headerBackgroundColor={HEADER_BACKGROUND_COLOR}
-        goBack={props.goBack}
-        headerTitle={I18n.t("bonus.cgn.name")}
-        dark={true}
-        titleColor={"white"}
-        contextualHelp={emptyContextualHelp}
-      >
-        <SafeAreaView style={IOStyles.flex}>
-          <FocusAwareStatusBar
-            backgroundColor={HEADER_BACKGROUND_COLOR}
-            barStyle={"light-content"}
-          />
-          {pot.isError(props.potCgnDetails) ? ( // subText is a blank space to avoid default value when it is undefined
-            <GenericErrorComponent
-              subText={" "}
-              onRetry={loadCGN}
-              onCancel={props.goBack}
+      <FocusAwareStatusBar
+        backgroundColor={HEADER_BACKGROUND_COLOR}
+        barStyle={"light-content"}
+      />
+      {pot.isError(props.potCgnDetails) ? ( // subText is a blank space to avoid default value when it is undefined
+        <GenericErrorComponent
+          subText={" "}
+          onRetry={loadCGN}
+          onCancel={navigation.goBack}
+        />
+      ) : (
+        <>
+          <Animated.ScrollView
+            contentContainerStyle={{
+              paddingBottom: safeBottomAreaHeight
+            }}
+            onScroll={scrollHandler}
+            scrollEventThrottle={16}
+            snapToOffsets={[0, headerHeight]}
+            snapToEnd={false}
+            decelerationRate="normal"
+          >
+            <View
+              style={{ height: 260, backgroundColor: HEADER_BACKGROUND_COLOR }}
             />
```

Comment:
```suggestion
          >
            {Platform.OS === "ios" && (
              <View
                style={{
                  position: "absolute",
                  height: 1000,
                  backgroundColor: HEADER_BACKGROUND_COLOR,
                  top: -1000,
                  right: 0,
                  left: 0
                }}
              />
            )}
            <View
              style={{ height: 260, backgroundColor: HEADER_BACKGROUND_COLOR }}
            />
```

What do you think if inside this scroll view, for iOS you also put a view that has the same color as the header? So as to improve the UX when scrolling too high on Apple devices.

|Before|After|
|-|-|
|<video src="https://github.com/pagopa/io-app/assets/34343582/773189f1-5eea-4160-8d86-3b0eede7bcde" />|<video src="https://github.com/pagopa/io-app/assets/34343582/40ced118-7a55-421e-be45-1ed9791a3287" />

Reply:
Instead of adding a view with arbitrary height and ui configs, what about disabling the bounces option of the ScrollView?


https://github.com/pagopa/io-app/assets/3959405/d5738e06-8490-49d6-a4d2-0fbbf16f436d

Reply:
commited in [78131ec](https://github.com/pagopa/io-app/pull/5651/commits/78131ec0640e722e829557e501c1805f234ea670)

Reply:
As an iOS user, I'm not a huge fan of the un-bounced scroll views but of course, it is more a point of view and preferences. So I think that it's up to you 👍

Reply:
I would avoid to add arbitrary height views just to avoid a different background on the top, so, but just my 2 cents:
- we leave the white color when overflowing the scroll
- we block the bounce
what do you think?

Reply:
That sounds good to me 👍